client/src/components/NavBar.jsx

```diff
@@ -63,8 +63,8 @@ function NavBar({ profile, updateAppProfile }) {
               Calendar
             </Link>
             <Link to="/message-wall" className={styles.btn_info}>
-              <img src={MessageIcon} alt="message icon" />
-              Messages
+              <img src={NoticesIcon} alt="message icon" />
```

Comment:
Since we're changing the icon and the text, we should probably also change the alt text to "notices icon"!